geode-core/src/distributedTest/java/org/apache/geode/internal/cache/execute/PRClientServerTestBase.java

```diff
@@ -99,10 +101,11 @@ public final void postSetUp() throws Exception {
     server2 = host.getVM(1);
     server3 = host.getVM(2);
```

Comment:
For future reference, these can be cleaned up to be
<Delete> Host host = Host.getHost(0);
server1 = VM.getVM(0);
server2 = VM.getVM(1);
server3 = VM.getVM(2);
client = VM.getVM(3);